modules/standard/IO.chpl

```diff
@@ -4040,13 +4059,21 @@ proc channel.writebits(v:integral, nbits:integral):bool throws {
   return try this.write(new ioBits(v:uint(64), nbits:int(8)));
 }
 
+/*
+   Write a binary number to the channel
+
+   :arg arg: number to be written
+   :arg endian: type:`ioendian` compile-time argument that specifies the byte order in which
+              to write the number. Defaults to `ioendian.native`.
```

Comment:
I think `ioendian.native` might need two ` on both ends.  Also the :type: tag is missing a colon at the beginning